### DISCUSSION TITLE
Daily 4/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#93] [General]
Asistí a la clase de inglés durante la mañana. Durante el día voy a realizar una de las tareas incluído diseño y maquetación. También sigo pensando cómo armar la tarea para el encabezado.

**Comment by NataliaAlvarezIspc** [comment]
Hoy no dispongo de mucho tiempo. Comenzare con la maquetación de una de las tareas en html y hare un pequeño repaso de SGC para la evaluación.

**Comment by Brian-Almada** [comment]
Ayer realicé el bosquejo para guiarme a la hora de construir el html de la página de contacto para miembros. Luego diseñé una idea de logo para Almoneda y revisé algo de SGC. Hoy voy a tratar de avanzar un poco más con SGC e intentar subir el html para terminar una de las issues.
